clusterlib/scheduler.py

```diff
@@ -254,7 +260,8 @@ def submit(job_command, job_name="job", time="24:00:00", memory=4000,
     if email_options:
         job_options.append(template["email_options"] % email_options)
 
-    if log_directory:
+    if log_directory is not None:
+        log_directory = os.path.abspath(log_directory)
```

Comment:
Otherwise I get SGE to fail to queue the job, telling us that the file "/home/travis/./cluster_lib_tmp_xxxx/logs" does not exists. It is apparently non-robust to relative path... SGE is really crappy compared to SLURM.